PilotDataPlatform/metadata
# PILOT-1628 Add bulk pin/unpin endpoint

## Summary

- Add a new endpoint for pinning/unpinning many favourites at once.

## JIRA Issues

https://indocconsortium.atlassian.net/browse/PILOT-1628

## Type of Change


## Testing

Are there any new or updated tests to validate the changes?


## Test Directions

1. Try pinning/unpinning multiple favourites with the new endpoint.
2. Check that the old, single pin/unpin endpoint still works as it changed a bit.
